pennylane/gradients/general_shift_rules.py

```diff
@@ -20,20 +22,26 @@
 import pennylane as qml
 
 
-def process_shifts(rule, tol=1e-10, check_duplicates=True):
+def process_shifts(rule, tol=1e-10, batch_duplicates=True):

```

Comment:
Curious how come you changed the name of this?

Reply:
I felt like `check_duplicates` is not as accurate a description as it could be, because it does not return a `bool` variable, a warning or an error, but it actually batches duplicate shifted terms. :)
I hope that this kwarg is new enough so that this change will not impact any users, what do you think?